cli/internal/cmd/configfetchmeasurements.go

```diff
@@ -78,9 +79,9 @@ func (cfm *configFetchMeasurementsCmd) configFetchMeasurements(
 	cfm.log.Debugf("Using flags %v", flags)
 
 	cfm.log.Debugf("Loading configuration file from %q", flags.configPath)
-	conf, err := config.New(fileHandler, flags.configPath)
+	conf, err := config.New(fileHandler, flags.configPath, flags.force)
 	if err != nil {
-		return displayConfigValidationErrors(cmd.ErrOrStderr(), err)
+		return config.DisplayValidationErrors(cmd.ErrOrStderr(), err)
```

Comment:
Have you considered to make the `--force` flag more broad? That is, it ignores any errors returned from config validation, keep print them as warnings to user, and then continue with execution.

I think most `--force` flags are a lot broader in what they ignore. I would expect this to be useful in situations where the user knows what they are doing, e.g., during testing, debugging, ...

Reply:
I shortly thought about it but came to the conclusion that it only really makes sense for version validation right now.

E.g. setting a VM type that is not supported is pretty likely to break things. On the other hand, setting weird version combinations is more likely to work. 

But I am happy to hear other opinions and change the flag to disable validation altogether.

Reply:
It is fine for me to keep the scope of `force` as is.